CHANGELOG.md

```diff
@@ -49,6 +49,7 @@
   * `s3.http.insecure-skip-verify`
 * [ENHANCEMENT] Prometheus upgraded. #2798 #2849
   * Optimized labels regex matchers for patterns containing literals (eg. `foo.*`, `.*foo`, `.*foo.*`)
+* [ENHANCEMENT] Add metric to Ruler to track failures of loading rules files. #2857 
```

Comment:
From what I see in other instance we tend to include the metric name in the changelog.

Reply:
Also please remove the trailing whitespace or run `make clean-white-noise` to fix it (otherwise the linter fails).